src/Ninja/G1CP/Content/Tests/test278.d

```diff
@@ -0,0 +1,39 @@
+/*
+ * #278 Ambient women dialog subtitles don't match the audio (EN)
+ */
+func int G1CP_Test_278() {
+    G1CP_Testsuite_CheckLang(G1CP_Lang_EN);
+    var int ouPtr; ouPtr = G1CP_Testsuite_CheckOu("SVM_16_NotNow");
+    G1CP_Testsuite_CheckPassed();
+
+    return G1CP_Testsuite_InspectOuString(ouPtr, "text", "Shh, you're not allowed to talk to us.");
+};
+
+
+
+//
+// Alternatively this. The idea is that it requires a manual inspection to compare subtitle text and spoken audio.
```

Comment:
Please let me know what you think about the question about manual or automatic test.

Reply:
I think it is sufficient to check this manually once and then go with the automatic tests.

Reply:
I agree. I will remove the commented out part soon. You can feel free to do it, too.